scripts/_helpers.py

```diff
@@ -595,7 +595,11 @@ def read_csv_nafix(file, **kwargs):
 def to_csv_nafix(obj, path, **kwargs):
     if "na_rep" in kwargs:
         del kwargs["na_rep"]
-    return obj.to_csv(path, **kwargs, na_rep=NA_VALUE)
+    if len(obj) > 0:
```

Comment:
better to use the property obj.isempty probably, good check!

Reply:
Agree that it would be great to have a more elegant emptiness check. But I was not able to found a way. The `empty` method belongs to the pandas dataframe and obj.empty throws an error if applied to a list while it's convenient to use an empty list for initialisation... 

According to a (quite hot) discussion on SO, an implicit boolean conversion  `if obj:` is recommended as the most pytonic way to test that an `obj` is non-empty. But that would throw an error if applied for a dataframe.

Reply:
Checked and found that there is always an explicit transformation to a dataframe befor to output even if the data structure was initialised as a list in the beginning. Have replaced the length condition with using the `empty` method and a bit modified an argument name to make more clear that the function is intended to be used on dataframes.

Reply:
Perfect, closing this